### ISSUE TITLE
Simai comments

### ISSUE THREAD

**Issue by MBmasher** [Neskol/MaiLib#44]
Comments in simai are denoted by  `||` - the comment marker and everything after it, until the next new line, should be ignored

**Comment by Neskol** [comment]
Didn't see the wiki specification for commenting, but it could be a handy feature, do you have exact specification for commenting (and/or other non-gameplay features)?

**Comment by MBmasher** [comment]
yeah it doesn't seem to be documented but the feature is recognised by the majdata editor and it's used in some charts
